### PR TITLE
chore: update copyright

### PR DESCRIPTION
**Pull request by hezhizhen**
Screenshot 2023-04-28 at 20 24 36

Use GoLand to automatically update them all.

### CODE REVIEW

File: cmd/envd-sshd/main.go [hunk 1]
```diff
@@ -1,5 +1,4 @@
-// Copyright 2022 The envd Authors
-// Copyright 2022 The okteto remote Authors
```
[the remark this answers is not in the document]
Comment (hezhizhen) [reply cmd/envd-sshd/main.go:2]: Thanks. I'll manually revert these deletions later.

[the remark this answers is not in the document]
Comment (hezhizhen) [reply cmd/envd-sshd/main.go:2]: How about the years? Update or not?

**Comment by hezhizhen** [comment]
@terrytangyuan PTAL

**Review by gaocegege — APPROVED** [review]

**Comment by hezhizhen** [comment]
One check failed. Please rerun it.

**Comment by hezhizhen** [comment]
> One check failed. Please rerun it.

@gaocegege @kemingy @terrytangyuan Thanks.

**Pull request merged**